components/checkbox/__tests__/type.test.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import Checkbox from '..';
+import Checkbox, { CheckboxRef } from '..';
```

Comment:
Import "CheckboxRef" is only used as types. (@typescript-eslint/consistent-type-imports)

Reply:
Fixed